public/kbn_sankey_vis_controller.js

```diff
@@ -44,7 +45,10 @@ module.controller('KbnSankeyVisController', function ($scope, $element, $rootSco
   };
 
   let _buildVis = function (data) {
-    data.slices =  data.slices ? data.slices : filterNodesAndLinks(data.slices.nodes, data.slices.links);
+    if(!resize){
+      data.slices=filterNodesAndLinks(data.slices.nodes, data.slices.links);
+      resize=false;
```

Comment:
This line is obsolete. The if statement already makes sure, that `resize` is `false`.

Reply:
Yes, you were right about it .